cypress/integration/finance/funds/funds.deleteBudget.spec.js

```diff
@@ -0,0 +1,90 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import NewFund from '../../../support/fragments/finance/funds/newFund';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import { getCurrentFiscalYearCode } from '../../../support/utils/dateTools';
+import { testType } from '../../../support/utils/tagTools';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+
+describe('ui-finance: Delete budget from fund', () => {
+  const currentBudgetSectionId = 'currentBudget';
+  const ledger = {
+    id: uuid(),
+    name: `autotest_ledger_${getRandomPostfix()}`,
+    code: `autotest_code_${getRandomPostfix()}`,
+    description: `autotest_ledger_ description_${getRandomPostfix()}`,
+    ledgerStatus: 'Frozen',
+    currency: 'USD',
+    restrictEncumbrance: false,
+    restrictExpenditures: false,
+    acqUnitIds: '',
+    fiscalYearOneId: ''
+  };
+
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getToken(Cypress.env('diku_login'), Cypress.env('diku_password'));
+
+    cy
+      .okapiRequest({
+        path: 'acquisitions-units/units',
+        limit: 1,
+      })
+      .then(({ body }) => {
+        ledger.acqUnitIds = [body.acquisitionsUnits[0].id];
+      });
+
+    cy
+      .okapiRequest({
+        path: 'finance/fiscal-years',
+        limit: 1,
+      })
+      .then(({ body }) => {
+        ledger.fiscalYearOneId = body.fiscalYears[0].id;
+      });
+  });
+
+  beforeEach(() => {
+    cy.createLedgerApi({
+      ...ledger
+    });
+  });
+
+  afterEach(() => {
+    cy.deleteLedgerApi(ledger.id);
+  });
+
+  it('C343211 should delete budget for a new fund', { tags: [testType.smoke] }, () => {
+    const defaultFund = { ...NewFund.defaultFund };
```

Comment:
why do you need class class NewFund ? can it be redesigned to default export?

Reply:
Again, same as below - I would like to do it in separate pull requests, since I followed this approach for all my test cases before and it will require changes in almost all thunderjet tests.

Reply:
I see usage of NewFund into 3 lines of code. It's quicker to apply this change than discuss it into review and manage such change additionally through new tasks)

Reply:
and in case of one test you can move code from beforeEach and afterEach into the test

Reply:
Updated